docker-compose.yml

```diff
@@ -71,7 +71,7 @@ services:
     command: npm run dev
 
   frontend:
-    image: node:14
+    image: node:18
```

Comment:
It have not been updated in the server docker setup?

Reply:
The `engine` is still running 14. It's only the `frontend` that has been updated. The deploy build step for the `frontend` is
```
docker run --rm --volume ${PWD}:/app -w="/app" node:18 npm install --omit-dev
docker run --rm --volume ${PWD}:/app -w="/app" node:18 ./node_modules/.bin/encore production
docker run --rm --volume ${PWD}:/app -w="/app" node:18 rm -rf node_modules
```